packages/react/CHANGELOG.md

```diff
@@ -18,23 +18,23 @@ All notable changes to this project will be documented in this file. See
 
 # 1.0.0-rc.7 (2020-10-25)
 
-**Note:** Version bump only for package @chakra-ui/core
+**Note:** Version bump only for package @chakra-ui/react
```

Comment:
I feel like these package references shouldn't change since it's historical data. I'm not even sure how this `CHANGELOG` is generated anyway.